[2024-04-23T11:21:18.899+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-04-23T11:21:18.904+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-04-23T11:21:18.904+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:21:18.915+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-15 00:00:00+00:00
[2024-04-23T11:21:18.919+0000] {standard_task_runner.py:60} INFO - Started process 76368 to run task
[2024-04-23T11:21:18.921+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-15T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmpriilxvz3']
[2024-04-23T11:21:18.922+0000] {standard_task_runner.py:88} INFO - Job 252: Subtask gen_dates
[2024-04-23T11:21:18.946+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:21:18.977+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:21:19.094+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T00:00:00+00:00'
[2024-04-23T11:21:19.095+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:21:21.609+0000] {python.py:202} INFO - Done. Returned value was: dates_20240315
[2024-04-23T11:21:21.625+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240315T000000, start_date=20240423T112118, end_date=20240423T112121
[2024-04-23T11:21:21.664+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:21:21.677+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T11:24:44.278+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-04-23T11:24:44.282+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [queued]>
[2024-04-23T11:24:44.283+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T11:24:44.293+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-03-15 00:00:00+00:00
[2024-04-23T11:24:44.297+0000] {standard_task_runner.py:60} INFO - Started process 81443 to run task
[2024-04-23T11:24:44.299+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'scheduled__2024-03-15T00:00:00+00:00', '--job-id', '274', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmphjh7126e']
[2024-04-23T11:24:44.299+0000] {standard_task_runner.py:88} INFO - Job 274: Subtask gen_dates
[2024-04-23T11:24:44.324+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates scheduled__2024-03-15T00:00:00+00:00 [running]> on host datalab-yasmine
[2024-04-23T11:24:44.358+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T11:24:44.476+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T00:00:00+00:00'
[2024-04-23T11:24:44.477+0000] {logging_mixin.py:188} INFO - {'start': '2024-03-15 00:00:00+00:00', 'end': '2024-04-23 11:24:44'}
[2024-04-23T11:24:44.477+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T11:24:46.957+0000] {python.py:202} INFO - Done. Returned value was: dates_20240315
[2024-04-23T11:24:46.973+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240315T000000, start_date=20240423T112444, end_date=20240423T112446
[2024-04-23T11:24:47.007+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T11:24:47.031+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
